plugins/backend/catalog-backend-module-okta/src/providers/OktaUserEntityProvider.ts

```diff
@@ -78,7 +78,7 @@ export class OktaUserEntityProvider extends OktaEntityProvider {
         },
         spec: {
           profile: {
-            displayName: user.profile.email,
+            displayName: user.profile.displayName,
```

Comment:
Is display guaranteed? Should it default to the email?

Do you think title of the entity should show the display name also?

Reply:
> Is display guaranteed? 

I believe the `displayName` is guaranteed to be present on the `profile` object:
https://github.com/okta/okta-sdk-nodejs/blob/master/src/types/models/UserProfile.d.ts#L28

My read from this [article](https://support.okta.com/help/s/article/User-display-name-does-not-have-a-value?language=en_US) is that the `displayName` defaults to `firstname lastname`, is user configurable, and will default to empty string if not properly configured. 

> Should it default to the email?

I don't have a strong preference, but out of simplicity I think it is fine for `displayName` to match between Okta and Backstage. Users can configure expressions in Okta to set `displayName` to the email by default if they need.

> Do you think title of the entity should show the display name also?

`displayName` is potentially modifiable by Okta users, and therefore can potentially contain garbage (ex. spaces, emojis, things that aren't identifiable). I think `email` is a better cannonicalization of the user at the moment.